v20.2/performance-benchmarking-with-tpc-c-10-warehouses.md

```diff
@@ -10,16 +10,20 @@ This page shows you how to reproduce [CockroachDB's TPC-C performance benchmarki
 <div class="filters filters-big clearfix">
   <button class="filter-button current"><strong>10</strong></button>
   <a href="performance-benchmarking-with-tpc-c-1k-warehouses.html"><button class="filter-button">1000</button></a>
-  <a href="performance-benchmarking-with-tpc-c-10k-warehouses.html"><button class="filter-button">10,000</button></a>
-  <a href="performance-benchmarking-with-tpc-c-100k-warehouses.html"><button class="filter-button">100,000</button></a>
+  <a href="performance-benchmarking-with-tpc-c-10k-warehouses.html"><button class="filter-button">13,000</button></a>
+  <a href="performance-benchmarking-with-tpc-c-100k-warehouses.html"><button class="filter-button">140,000</button></a>
 </div>
 
-Warehouses | Data size | Cluster size
------------|-----------|-------------
-10 | 2GB | 3 nodes on your laptop
-1000 | 80GB | 3 nodes on `c5d.4xlarge` machines
-10,000 | 800GB | 15 nodes on `c5d.4xlarge` machines
-100,000 | 8TB | 81 nodes on `c5d.9xlarge` machines
+<strong><font style="size:xx-large" color="red">FIXME</font></strong>: Update data size column in table below
+
+| Cluster size                       | Data size | Warehouses |
```

Comment:
This is me switching to tables that focus on "Cluster size -> result" ordering as we discussed.  I did this on all the TPC-C pages.

Reply:
What should the 'Data size' columns be for the various runs?  Does anything need to be updated?  Presumably 13k and 140k means more data.